app/Http/Controllers/Api/InventoryController1.php

```diff
@@ -0,0 +1,51 @@
+<?php namespace App\Http\Controllers;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Item;
+
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\Input;
+
+class InventoryController1 extends Controller {
+
+	/**
+	 * Display a listing of the resource.
+	 *
+	 * @param  string $query
+	 * @return json all or queried $items
+	 */
+	public function index($query)
```

Comment:
What we want this endpoint to do is return a list of all inventory items, filtered by some string. The user passes that string via [a query string](http://en.wikipedia.org/wiki/Query_string) parameter like `http://192.168.33.99/api/inventory?query=blah`.

Look at [this StackOverflow question](http://stackoverflow.com/questions/24744825/laravel-queries-strings) for how to get access to a request's query string parameters.